core/client/hdfs/src/test/java/alluxio/hadoop/AbstractFileSystemTest.java

```diff
@@ -123,11 +125,14 @@ public void before() throws Exception {
   @After
   public void after() {
     HadoopClientTestUtils.resetClient();
+    // Zookeeper configuration may be set through Alluxio on Zookeeper URI
```

Comment:
`ConfigurationTestUtils.resetConfiguration()` could be useful here

Reply:
Changed, thanks!